README.md

```diff
@@ -100,7 +109,34 @@ Each lion should have...
 Create a new lion instance with the name "Simba".
 
 ```ruby
-# code here
+# class Animal
+#   attr_accessor :name
+#
+#   def initialize (name)
+#     @name = name
+#   end
+#
+#   def greet (name)
+#     puts "Hello, #{@name}"
+#   end
+# end
+#
+# pumba=  Animal.new("Pumba")
+#
+# class Lion < Animal
+#   attr_accessor :name
+#
+#   def initialize (name)
+#     @name = name
+#   end
+#
+#   def greet (name)
+#     puts "Hello, #{@name}"
+#   end
+#
+# end
+#
+# simba = Lion.new("Simba")
```

Comment:
Good start. A few things...
- You're missing the attributes that make a `Lion` different from an `Animal`
- Because `Lion` inherits from `Animal`, you don't need to replicate information (e.g., no need to redefine `greet`)